tests/cql/CqlListOperatorsTest.xml

```diff
@@ -154,7 +174,7 @@
 	</group>
 	<group name="Except">
 		<test name="ExceptEmptyListAndEmptyList">
-			<expression>Except({}, {})</expression>
+			<expression>{} except {}</expression>
```

Comment:
I don't think `Except({}, {})` is a valid way to represent the `except` operator.  CQL-to-ELM translates it as a `FunctionRef`.